src/App.js

```diff
@@ -18,6 +22,10 @@ function App() {
       <Logo />
       <div className="App">
         {/* STEP 4 - Render your components here and be sure to properly import/export all files */}
+        <Numbers/>
+        <Operators/>
+        <Specials/>
+        <Display/>
```

Comment:
This file is perfect. We import all our components in here, and then render them inside of our JSX